### PR TITLE
Re-enable scalajs support

### LINKED ISSUE #701: ScalaJS artifacts for Scala 2.13 and Scala 3
I cannot find scalajs artifacts for Scala 2.13 and Scala 3:

```
➜  myproject cs complete com.47deg:github4s_sjs
github4s_sjs0.6_2.10
github4s_sjs0.6_2.11
github4s_sjs0.6_2.12
```

### PR DESCRIPTION
**Pull request by vincenzobaz**
Fixes #701
Todo: set up publishing for scala js artifact

### CODE REVIEW

**Comment by fedefernandez** [comment]
@vincenzobaz please run `scalafmtAll` and `scalafmtSbt` before committing the changes. Thanks

File: build.sbt [hunk 1]
```diff
@@ -10,13 +10,18 @@ val allScalaVersions = scala2Versions :+ scala3Version
 ThisBuild / scalaVersion       := scala213
 ThisBuild / crossScalaVersions := allScalaVersions
 
-addCommandAlias("ci-test", "scalafmtCheckAll; scalafmtSbtCheck; mdoc; ++test")
+addCommandAlias(
+  "ci-test",
+  "scalafmtCheckAll; scalafmtSbtCheck; documentation/mdoc; ++test; ++github4sJS/test"
```
Comment (fedefernandez) [inline build.sbt:15]: If I'm not wrong, `++test` will execute the task for all the modules

**Review by fedefernandez — COMMENTED** [review]
Thanks for the PR! I'm missing some configurations from the build before the support was dropped, like :
```scala
lazy val github4sJVM = github4s.jvm
lazy val github4sJS  = github4s.js
```

This is the file before #309

File: build.sbt (hunk 1 above)
Comment (vincenzobaz) [reply build.sbt:15]: I did not know this notation.
Then if we the output twice in the ci, we can remove `++github4sJS/test`

Comment (fedefernandez) [reply build.sbt:15]: `++test` will run the test for all the supported scala versions plus scalajs. no need to add `++github4sJS/test` :)

File: build.sbt [hunk 2]
```diff
@@ -10,13 +10,15 @@ val allScalaVersions = scala2Versions :+ scala3Version
 ThisBuild / scalaVersion       := scala213
 ThisBuild / crossScalaVersions := allScalaVersions
 
-addCommandAlias("ci-test", "scalafmtCheckAll; scalafmtSbtCheck; mdoc; ++test")
+addCommandAlias("ci-test", "scalafmtCheckAll; scalafmtSbtCheck; documentation/mdoc; ++test")
```
Comment (fedefernandez) [inline build.sbt:13]:
```suggestion
addCommandAlias("ci-test", "scalafmtCheckAll; scalafmtSbtCheck; mdoc; ++test")
```

**Review by juanpedromoreno — APPROVED** [review]

**Review by fedefernandez — APPROVED** [review]

**Comment by juanpedromoreno** [comment]
Thanks @vincenzobaz !

**Pull request merged**